### PR TITLE
219 test 

### PR DESCRIPTION
**Pull request by Future-Genius**
Complete #219 #182

### CODE REVIEW

**Comment by kathrynkodama** [comment]
Please indicate which issues this pull request addresses

File: jakarta.jdt/org.eclipse.lsp4jakarta.jdt.test/src/main/java/org/eclipse/lsp4jakarta/jdt/annotations/PostConstructAnnotationTest.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+import org.eclipse.lsp4j.TextEdit;
+import org.eclipse.lsp4jakarta.commons.JakartaDiagnosticsParams;
+import org.eclipse.lsp4jakarta.commons.JakartaJavaCodeActionParams;
+import org.eclipse.lsp4jakarta.jdt.core.BaseJakartaTest;
+import org.eclipse.lsp4jakarta.jdt.core.JDTUtils;
+import org.junit.Test;
+
+public class PostConstructAnnotationTest extends BaseJakartaTest{
+	
+	protected static JDTUtils JDT_UTILS = new JDTUtils();
```
Comment (kathrynkodama) [inline jakarta.jdt/org.eclipse.lsp4jakarta.jdt.test/src/main/java/org/eclipse/lsp4jakarta/jdt/annotations/PostConstructAnnotationTest.java:27]: formatting is off

File: jakarta.jdt/org.eclipse.lsp4jakarta.jdt.test/src/main/java/org/eclipse/lsp4jakarta/jdt/annotations/PostConstructAnnotationTest.java [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    public void GeneratedAnnotation() throws Exception {
+        IJavaProject javaProject = loadJavaProject("jakarta-sample", "");
+        IFile javaFile = javaProject.getProject()
+                .getFile(new Path("src/main/java/io/openliberty/sample/jakarta/annotations/PostConstructAnnotation.java"));
+        String uri = javaFile.getLocation().toFile().toURI().toString();
+
+        JakartaDiagnosticsParams diagnosticsParams = new JakartaDiagnosticsParams();
+        diagnosticsParams.setUris(Arrays.asList(uri));
+
+        // expected annotations
```
Comment (kathrynkodama) [inline jakarta.jdt/org.eclipse.lsp4jakarta.jdt.test/src/main/java/org/eclipse/lsp4jakarta/jdt/annotations/PostConstructAnnotationTest.java:39]: comment should say "// expected diagnostics"

Comment (Future-Genius) [reply jakarta.jdt/org.eclipse.lsp4jakarta.jdt.test/src/main/java/org/eclipse/lsp4jakarta/jdt/annotations/PostConstructAnnotationTest.java:39]: I will solve this later today

File: jakarta.jdt/org.eclipse.lsp4jakarta.jdt.test/src/main/java/org/eclipse/lsp4jakarta/jdt/annotations/PostConstructAnnotationTest.java (hunk 1 above)
Comment (Future-Genius) [reply jakarta.jdt/org.eclipse.lsp4jakarta.jdt.test/src/main/java/org/eclipse/lsp4jakarta/jdt/annotations/PostConstructAnnotationTest.java:27]: solved

File: jakarta.jdt/org.eclipse.lsp4jakarta.jdt.test/src/main/java/org/eclipse/lsp4jakarta/jdt/annotations/PostConstructAnnotationTest.java (hunk 2 above)
Comment (Future-Genius) [reply jakarta.jdt/org.eclipse.lsp4jakarta.jdt.test/src/main/java/org/eclipse/lsp4jakarta/jdt/annotations/PostConstructAnnotationTest.java]: solved

File: jakarta.jdt/org.eclipse.lsp4jakarta.jdt.test/src/main/java/org/eclipse/lsp4jakarta/jdt/annotations/PostConstructAnnotationTest.java [hunk 3]
```diff
@@ -0,0 +1,61 @@
+package org.eclipse.lsp4jakarta.jdt.annotations;
```
Comment (kathrynkodama) [inline jakarta.jdt/org.eclipse.lsp4jakarta.jdt.test/src/main/java/org/eclipse/lsp4jakarta/jdt/annotations/PostConstructAnnotationTest.java:1]: This class is missing a header

File: jakarta.jdt/org.eclipse.lsp4jakarta.jdt.test/src/main/java/org/eclipse/lsp4jakarta/jdt/annotations/PostConstructAnnotationTest.java [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+import org.eclipse.lsp4j.Diagnostic;
+import org.eclipse.lsp4j.DiagnosticSeverity;
+import org.eclipse.lsp4j.TextEdit;
+import org.eclipse.lsp4jakarta.commons.JakartaDiagnosticsParams;
+import org.eclipse.lsp4jakarta.commons.JakartaJavaCodeActionParams;
+import org.eclipse.lsp4jakarta.jdt.core.BaseJakartaTest;
+import org.eclipse.lsp4jakarta.jdt.core.JDTUtils;
+import org.junit.Test;
+
+public class PostConstructAnnotationTest extends BaseJakartaTest {
```
Comment (kathrynkodama) [inline jakarta.jdt/org.eclipse.lsp4jakarta.jdt.test/src/main/java/org/eclipse/lsp4jakarta/jdt/annotations/PostConstructAnnotationTest.java:25]: Add a javadoc comment describing the purpose of this test

**Review by kathrynkodama — COMMENTED** [review]
Looks good thanks @Future-Genius, just a few minor comments

**Review by kathrynkodama — APPROVED** [review]

**Pull request merged**